README.txt

```diff
@@ -1,21 +1,24 @@
-Methodocracy
+Methodocracy (noun):
+A cultural-political ideology which proposes that optimal decision-making, from the individual to national level, is necessarily based upon the quantity and quality of data, interpretation, and conclusions gathered through tested and reliable methodologies.
 
-"The Super-Power of Problem-Solving"
 
+Hurdles: 
+1. It is prohibitively expensive and time-consuming for the average citizen to access information due to the high cost of scientic journals/articles and the lack of a centralized database.
 
-A free website that facilitates problem-solving. Kind of like Wikipedia, but instead of logging already known information, it focuses on accumulating new knowledge using established methods, especially the scientific method. Credentialed users and average Joe's can use the site alike.
+2. Even with free and convenient access to a centralized information archive, the vast majority of the population have little training nor ability to determine the quality of data, interpretations, conclusions, and methodologies. This results in confusion and distrust for information-based-decision-making, since bad information contradicts good information and the two are indistinguishable to the untrained mind.
 
 
-We need to work together to make unlikely connections, to think more creatively as a whole than we would have alone.
+Proposal:
+To create and develop a free, user-friendly, crowdsourced website that archives and rates research projects in order to facilitate problem-solving and decision-making. A chimera of wikipedia, researchgate, and reddit, it would allow users to submit questions/problems, reply with hypotheses or entire researh projects, and rate others' submissions on scientific coherence and readability. It will focus on teaching and guiding its user on data collection, analysis, and interpretation, while acting as a database for all user-submitted queries and researches. Additionally, it would serve as a scientific-networking platform for both small and large scale collaborations. All users will be able to rate all submissions, however the impact of a user's rating will be based upon the quality/quantity of their contributions, their work/education background, and other relevant factors.
 
 
-Benefits
+Goals:
 
-    Centralizes scientific publication.
-    Organizes knowledge robustly.
-    Takes advantage of lower merit knowledge in addition to higher merit knowledge.
-    Open, free, and free from special interests.
-    Easy to cite.
+    To centralize scientific publications.
+    To teach, enable, and engage the entire population in problem solving through the scientific method.
+    To robustly organize and rank information.
```

Comment:
Ranking CAN be part of organization, it is not necessarily so. You can organize a photo album without considering how much you like the picture and go solely on chronological order or whatever other criteria.